gspread_formatting/models.py

```diff
@@ -187,11 +189,29 @@ def __init__(self, themeColor=None, rgbColor=None):
 class Color(CellFormatComponent):
     _FIELDS = ('red', 'green', 'blue', 'alpha')
 
-    def __init__(self, red=None, green=None, blue=None, alpha=None):
+    def __init__(self, red=None, green=None, blue=None, alpha=None, hexcolor=None):
         self.red = red
         self.green = green
         self.blue = blue
         self.alpha = alpha
+        if hexcolor:
+            self.fromHex(hexcolor)
+
+    def toHex(self):
+        # Dont worry about alpha channel yet
+        # This would simpler if the default was 0 instead of None
+        RR = format(floor((self.red if self.red else 0) * 255), '02x')
+        GG = format(floor((self.green if self.green else 0) * 255), '02x')
+        BB = format(floor((self.blue if self.blue else 0) * 255), '02x')
+        hexformat = f'#{RR}{GG}{BB}'
+        return hexformat
+
+    def fromHex(self,hexcolor):
```

Comment:
This should be a `classmethod`, so that one can call `Color.fromHex(some_string)` and receive a `Color` object in return:

```
@classmethod
def fromHex(cls, hexcolor):
    return cls(int(hexcolor[1:3],16) / 255, ...)